homeplug_exporter.go

```diff
@@ -131,6 +133,7 @@ func (n *HomeplugNetworkInfo) UnmarshalBinary(b []byte) error {
 			return err
 		}
 		n.Networks = append(n.Networks, ns)
+		level.Debug(logger).Log("msg", fmt.Sprintf("Network found: %s", &ns))
```

Comment:
For structured (i.e., machine-readable) logs, this works better:
```suggestion
		level.Debug(logger).Log("msg", "Network found", "network", &ns)
```

Reply:
ahh, I wondered sometimes why logs were formatted like this